Cargo.toml

```diff
@@ -3,7 +3,11 @@ members = ["rcgen", "rustls-cert-gen"]
 resolver = "2"
 
 [workspace.dependencies]
-pem = { version = "3.0.2" }
+pem = "3.0.2"
+rand = "0.8"
+ring = "0.17"
+rsa = "0.9"
```

Comment:
[My feedback](https://github.com/rustls/rcgen/pull/185/files#r1385579137) on the other PR about the `rsa` dependency was marked as resolved, but the dependency is still present. 

Do you disagree with my suggestion we consider RSA key generation out-of-scope for this tool since rcgen the library can't support it, or did something get lost in the shuffle?

Reply:
It looks like I removed it from the `rustls-cert-gen` crate but forgot to remove it here.